install/kubernetes/cilium/templates/clustermesh-apiserver/deployment.yaml

```
@@ -161,6 +161,7 @@ spec:
         - --enable-external-workloads={{ .Values.externalWorkloads.enabled }}
         {{- if .Values.clustermesh.apiserver.metrics.enabled }}
         - --prometheus-serve-addr=:{{ .Values.clustermesh.apiserver.metrics.port }}
+        - --controller-group-metrics all
```

```suggestion
        - --controller-group-metrics=all
```

This should fix the currently failing tests. Same for the occurrence below.